safemysql.class.php

```diff
@@ -616,6 +620,20 @@ private function escapeInt($value)
 		return $value;
 	}
 
+	private function escapeBool($value)
+	{
+		if ($value === true || $value === 1 || $value === '1')
```

Comment:
I don't understand how the third one could happen. As in, do we want to silently cast to bools?